.github/workflows/create-release-pr.yml

```diff
@@ -0,0 +1,127 @@
+name: Create Release PR
+
+on:
+  workflow_dispatch:
+    inputs:
+      source_ref:
+        description: 'Source branch, commit, or tag to create the release branch from'
+        default: 'develop'
+        type: string
+      release_version:
+        description: 'In the vX.X.X format and will be used for tag and branch name'
+        default: 'vx.x.x'
+        required: true
+        type: string
+
+env:
+  RELEASE_VERSION: ${{ inputs.release_version }}
+  SOURCE_REF: ${{ inputs.source_ref }}
+
+jobs:
+  create-release-pr:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ inputs.source_ref }}
+
+      - name: Create Release Branch
+        id: create-release-branch
+        run: |
+          if [[ $RELEASE_VERSION =~ ^v[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+            git checkout -b release/$RELEASE_VERSION
+            git push --set-upstream origin release/$RELEASE_VERSION
+          else
+            echo "Version input doesn't match the regex pattern ^v[0-9]+\.[0-9]+\.[0-9]+$"
+            exit 1
+          fi
+          echo "release-version=$RELEASE_VERSION" >> $GITHUB_OUTPUT
+          echo "release-branch=release/$RELEASE_VERSION" >> $GITHUB_OUTPUT
+
+      - name: Get Latest Release
+        id: get-release
+        run: |
+          releases=$(curl --header "Authorization: Bearer ${{ secrets.GITHUB_TOKEN }}" \
+                      --header "Accept: application/vnd.github.v3+json" \
+                      "https://api.github.com/repos/${{ github.repository }}/releases")
+          tag=$(echo "$releases" | jq -r '.[0].tag_name')
+          echo "latest-tag=$tag" >> $GITHUB_OUTPUT
```

Comment:
Will the above only get published releases, not pre-releases? Or maybe we want that for comparison?

Reply:
Updated with the releases/latest api and tested it here: https://github.com/algorand/py-algorand-sdk/commit/f064fa2d9f47580f0f1b531924fcb1b8b878e9a1 (will delete this branch later)